docs/upgrade.html

```diff
@@ -19,6 +19,12 @@
 
 <script id="upgrade-template" type="text/x-handlebars-template">
 
+<h5><a id="upgrade_260_notable" href="#upgrade_260_notable">Notable changes in 2.6.0</a></h5>
+<ul>
+    <li>Kafka Streams adds a new processing mode (requires broker 2.5 or newer) that improves application
```

Comment:
Do we need to attach a link to KIP here as well?

Reply:
I don't think we _need_ but sound like a good idea :)